pages/settings/account.vue

```diff
@@ -66,53 +271,83 @@
-        <button type="button" class="iconified-button" value="Copy to clipboard" @click="copyToken">
-          <template v-if="copied">
-            <CheckIcon />
-            Copied token to clipboard
-          </template>
-          <template v-else> <CopyIcon />Copy token to clipboard </template>
-        </button>
-        <button type="button" class="iconified-button" @click="$refs.modal_revoke_token.show()">
-          <SlashIcon />
-          Revoke token
-        </button>
+      <div class="adjacent-input">
+        <label for="theme-selector">
+          <span class="label__title">Email</span>
+          <span class="label__description">Changes the email associated with your account.</span>
+        </label>
+        <div>
+          <button class="iconified-button" @click="$refs.changeEmailModal.show()">
+            <template v-if="auth.user.email">
+              <EditIcon />
+              Change email
+            </template>
+            <template v-else>
+              <PlusIcon />
+              Add email
+            </template>
+          </button>
+        </div>
+      </div>
+      <div class="adjacent-input">
+        <label for="theme-selector">
+          <span class="label__title">Password</span>
+          <span v-if="auth.user.has_password" class="label__description">
+            Change <template v-if="auth.user.auth_providers.length > 0">or remove</template> the
+            password used to login to your account.
+          </span>
+          <span v-else class="label__description">
+            Set a permanent password to login to your account.
+          </span>
+        </label>
+        <div>
+          <button
+            class="iconified-button"
+            @click="
+              () => {
+                oldPassword = ''
+                newPassword = ''
+                confirmNewPassword = ''
+                removePasswordMode = false
+                $refs.managePasswordModal.show()
+              }
+            "
+          >
+            <KeyIcon />
+            <template v-if="auth.user.has_password"> Change password </template>
+            <template v-else> Add password </template>
+          </button>
+        </div>
+      </div>
+      <div class="adjacent-input">
+        <label for="theme-selector">
+          <span class="label__title">Two-factor authentication</span>
+          <span class="label__description">
+            Add an additional layer of security to your account during login.
+          </span>
+        </label>
+        <div>
+          <button class="iconified-button" @click="showTwoFactorModal">
+            <template v-if="auth.user.has_totp"> <TrashIcon /> Remove 2FA </template>
+            <template v-else> <PlusIcon /> Setup 2FA </template>
+          </button>
+        </div>
+      </div>
+      <div class="adjacent-input">
+        <label for="theme-selector">
+          <span class="label__title">Manage authentication providers</span>
```

Comment:
Button already says manage, I'd just have the title here be Authentication providers